mtp_noms_ops/apps/views.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+from django.urls import reverse_lazy
+from django.utils.translation import gettext_lazy as _
+from django.views.generic import TemplateView
+
+
+class FAQView(TemplateView):
+    template_name = 'faq.html'
+    title = _('What do you need help with?')
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context['breadcrumbs_back'] = reverse_lazy('login')
+        context['cashbook_url'] = settings.CASHBOOK_URL
+        context['reset_password_url'] = reverse_lazy('reset_password')
+        # TODO: Functionality not there yet.
+        # context['sign_up_url'] = reverse_lazy('sign-up')
+        context['sign_up_url'] = 'https://example.com/TODO-sign-up-url'
```

Comment:
in the short term and if you wanted this to be releasable, you could just point this to the contact us page with a pre-filled message. for example `I want to request an account. [Please provide your name, email address and Quantum ID]`. that way, you don't depend on the sign-up page and all that'd need changing after that's done is this one link.

Reply:
Great advice as always Igor, thanks! That seems sensible, I'll do so that this is not blocked and we can add the link later 👍 💯  (I was actually wondering if I should comment that Q/A out for the time being but your suggestion is probably even better)

Reply:
FYI: Update code as suggested.